utils/config_manager.py

```
@@ -352,9 +360,13 @@ def flatten_reserved(catgirl_data: dict) -> dict:
     if vrm_animation is not None:
         result["vrm_animation"] = vrm_animation
 
-    idle_animation = get_reserved(result, "avatar", "vrm", "idle_animation", default="")
+    idle_animation = get_reserved(result, "avatar", "vrm", "idle_animation", default=[])
     if idle_animation:
-        result["idleAnimation"] = idle_animation
+        # 向前兼容：确保输出为 list
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)